src/components/Header.jsx

```diff
@@ -3,12 +3,16 @@ import { Link } from "react-router-dom";
 import pageBanner from "../assets/images/pageBanner.png";
 
 const Header = () => {
+
   return (
     <div>
```

Comment:
Is there a more semantic element than a `div` you could use to separate out the header? 👀